internal/services/mssql/mssql_server_dns_alias_resource.go

```diff
@@ -0,0 +1,143 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
```

Comment:
There shouldn't import this package, all the exported entries are wrapped by `pluginsdk` instead.